rollup.config.mjs

```diff
@@ -4,20 +4,33 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
+import {getBabelOutputPlugin} from '@rollup/plugin-babel';
 import commonjs from '@rollup/plugin-commonjs';
 import resolve from '@rollup/plugin-node-resolve';
 import replace from '@rollup/plugin-replace';
+import process from 'process';
 import postcss from 'rollup-plugin-postcss';
 import summary from 'rollup-plugin-summary';
 import {terser} from 'rollup-plugin-terser';
 import ts from 'rollup-plugin-ts';
 
+const devMode = process.env.NODE_ENV === 'development';
+console.log(`${devMode ? 'development' : 'production'} mode bundle`);
+
```

Comment:
very basic logging for `NODE_ENV`. It makes it easier to tell which type of bundle is being served during development.